### PR TITLE
Add initial definitions for CD terms 📙

### PR DESCRIPTION
**Pull request by bobcatfish**
The discussion about these definitions was kicked off in the
interoperability working group, then moved to
[this google doc](https://docs.google.com/document/d/1IUSmgtw5eC2JwyfiX7IPK3twl3MSjWAsmCKmJ6Y5z-w/edit#)
and now is being moved to this repo, after having established (at least
an initial) timeline in timeline.md of when these terms came into being.

This is intended to just be the first iteration of these defintions and
to be refined over time.

@tracymiranda @salaboy @tequilarista @fdegir

### CODE REVIEW

File: definitions.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* [Continuous Integration](#continuous-integration)
+* [CI/CD](#cicd)
+
+## Continuous Delivery
+
+**Continuous Delivery (CD)**: A software development practice in which teams release software changes to users safely,
+quickly and sustainably by:
+
+ 1. Proving that changes can be released at any time
+ 2. Automating release processes
```
Comment (bobcatfish) [inline definitions.md:20]: @tracymiranda i tried to capture both your more succinct definition AND these 2 points that i think are important but i can drop these 2 if you feel strongly 🙏

File: definitions.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* [Continuous Delivery](#continuous-delivery): Continuous Deployment is made possible by Continuous Delivery -
+  [Jez Humble](https://continuousdelivery.com/2010/08/continuous-delivery-vs-continuous-deployment/),
+  [Timothy Fitz](https://www.blazemeter.com/blog/five-things-you-should-know-about-continuous-deploymentby-man-who-coined-term),
+  [Martin Fowler](https://martinfowler.com/bliki/ContinuousDelivery.html)
+* [Continuous Integration](#continuous-integration): Continuous Integration is a requirement for Continuous Deployment -
+  [Timothy Fitz](http://timothyfitz.com/2012/11/25/paths-to-continuous-deployment/)
+
+## Continuous Integration
+
+**Continuous Integration (CI)**: The process of combining code changes frequently, where each change (release candidate)
```
Comment (tequilarista) [inline definitions.md:50]: I'm not sure the "(release candidate)" part is useful here.  There could be legit policy reasons for not having a one to relationship between a commit and a deploy, but that the other elements (automatically validated, etc.) still very much apply.

Comment (tequilarista) [reply definitions.md:50]: Also, per Terry Cox we should refer to explicit methodologies that can be compared against

Comment (bobcatfish) [reply definitions.md:50]: > Also, per Terry Cox we should refer to explicit methodologies that can be compared against

@tequilarista can you explain a bit more?

**Pull request merged**